src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -235,7 +234,7 @@ protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFact
                 }
 
                 dependencyList.Add(factory.VTable(_type), "VTable");
-                
```

Comment:
Please can you clean up all these white space changes? It makes it hard to see the real changes.